tests/kola/version/rhaos-pkgs-match-openshift

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 ## kola:
 ##   exclusive: false
-#
-# This is RHCOS only
+##   description: Verify that rhaos packages match the OpenShift 
```

Comment:
Small typo: `rhaos` -> `rhcos`

Reply:
Ok, `rhaos` is valid - learning every day :D 

Sorry for the above!

Reply:
Both are OK to me, but I would keep `rhaos` to be consistent in later `rpm -qa | grep rhaos`

Reply:
RHAOS -> Red Hat Atomic OS, old name for RHCOS.